pkg/services/dashboards/database/database_dashboard_public_test.go

```diff
@@ -237,9 +255,17 @@ func TestIntegrationUpdatePublicDashboard(t *testing.T) {
 		pdRetrieved, err := dashboardStore.GetPublicDashboardConfig(context.Background(), savedDashboard.OrgId, savedDashboard.Uid)
 		require.NoError(t, err)
 
+		pdNotUpdatedRetrieved, err := dashboardStore.GetPublicDashboardConfig(context.Background(), anotherSavedDashboard.OrgId, anotherSavedDashboard.Uid)
```

Comment:
NitPick. maybe move line 258 and 259 to 266. I think it'll be easier to read when we're retrieving and asserting at the same time.